packages/strapi-plugin-content-manager/admin/src/components/Inputs/index.js

```diff
@@ -106,6 +92,28 @@ function Inputs({ autoFocus, keys, layout, name, onBlur }) {
     );
   }
 
+  if (type === 'uid') {
```

Comment:
You can pass the `InputUID` directly to the `Inputs` so you don't need this statement.
To do so you need to add the `uid` case to the `getInputType` and add the `contentTypeUID` props to the `Inputs`

Reply:
I tried to do that before but I got an error about the controlled component because the value is null at the initialization. The value passed to the Inputs component is `const value = get(modifiedData, keys, null);`. Like the relation input, I just need `get(modifiedData, keys)`.

Reply:
Ok leave it this way then